.changeset/two-tables-do.md

```diff
@@ -4,7 +4,7 @@
 "@salt-ds/theme": minor
 ---
 
-Removed differential characteristic
+Deprecated differential characteristic
```

Comment:
```suggestion
Deprecated differential characteristic: use `--salt-status-positive-foreground`, `--salt-status-negative-foreground` instead of `--salt-differential-positive-foreground`, `--salt-differential-negative-foreground`.
```